src/main/java/frc/robot/commands/AutoShoot.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.Constants;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.GoalCamera;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Shooter;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class AutoShoot extends CommandBase {
+
+  private Drivetrain drivetrain;
+  private GoalCamera goalCamera;
+  private double robotVelocity;
+  private Command runIntake;
+  private Command runFlywheel;
```

Comment:
The only command that should be an object variable is the command group that will be scheduled in the initialize method.